x/bank/keeper/migrations.go

```diff
@@ -25,3 +26,11 @@ func (m Migrator) Migrate1to2(ctx sdk.Context) error {
 func (m Migrator) Migrate2to3(ctx sdk.Context) error {
 	return v046.MigrateStore(ctx, m.keeper.storeKey, m.keeper.cdc)
 }
+
+// Migrate3to4 migrates x/bank storage from version 3 to 4.
+func (m Migrator) Migrate3to4(ctx sdk.Context) error {
+	oldParams := m.keeper.GetParams(ctx)
+	m.keeper.SetAllSendEnabled(ctx, oldParams.GetSendEnabled())
+	m.keeper.SetParams(ctx, banktypes.NewParams(oldParams.DefaultSendEnabled))
```

Comment:
Why are we calling this line?

Reply:
It is deprecated -- that's the point of this feature. Deprecate the `DefaultSendEnabled` and use a more strongly structured list. In fact, the Proto schema states it's deprecated.